### PR TITLE
Feature/load testing

### PR DESCRIPTION
**Pull request by shamoons**
Make sure to merge #2 first

### CODE REVIEW

File: load-testing/secret.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (gjanakiram) [inline load-testing/secret.json]: Is this load test case ? Are we going to run this from our application or we will be using tool to load test ?

File: load-testing/secret.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "config": {
+    "target": "http://localhost:3001",
```
Comment (gopalakrishnanu) [inline load-testing/secret.json]: Can the target be overriden from command line when running the test?

File: load-testing/secret.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "phases": [
+      {
+        "duration": 10,
+        "arrivalRate": 5
+      }
+    ],
+    "variables": {
+      "name": "shamoon",
+      "password": "password"
```
Comment (gopalakrishnanu) [inline load-testing/secret.json]: is there a way to externalize sensitive info like this one?

File: load-testing/secret.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "scenarios": [
+    {
+      "flow": [
+        {
+          "post": {
+            "url": "/login",
+            "json": {
+              "name": "{{ name }}",
+              "password": "{{ password }}"
+            }
```
Comment (gopalakrishnanu) [inline load-testing/secret.json]: Please extend the scenario by invoking the secured url after the login.

File: load-testing/secret.json (hunk 2 above)
Comment (gopalakrishnanu) [reply load-testing/secret.json]: Also a README to the load-testing folder with instructions on running the test.

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,3 @@
+[*.js]
```
Comment (gopalakrishnanu) [inline .editorconfig]: Please add to .gitignore

File: test/auth.js [hunk 6]
```diff
@@ -10,5 +12,23 @@ describe("Basic Authentication with JWT", () => {
...
       .expect(200)
-  });
+      .then((res) => {
+        jwt = res.body.token
+        assert(res.body.token.length > 0, true)
+      })
+
+  })
+
+  it('Should not allow access to the secret route without a token', () => {
```
Comment (gopalakrishnanu) [inline test/auth.js]: Looks good

**Pull request merged**